src/core/annotations/field-value.ts

```diff
@@ -0,0 +1,38 @@
+import {isPresent} from 'angular2/src/facade/lang';
+
+
+declare var Symbol: any;
```

Comment:
Is this legal? TS already supports ES6 `Symbol` for built-in symbols, so wouldn't there be a conflict? Why use Symbol at all? I'm not sure I see the benefit.